packages/rest/lib/http.cadl

```diff
@@ -9,13 +9,18 @@ model Response<Status> {
   @statusCode
   statusCode: Status;
 }
-model OkResponse<Body> is Response<200> {
-  @doc("The reponse body.")
+
+/**
+ * Defines a model with a single property of the given type, marked with `@body`.
+ *
+ * This can be useful in situations where you cannot use a bare T as the body
+ * and it is awkward to add a property.
+ */
+model Body<T> {
```

Comment:
Would `Payload<T>` be more obvious or even less obvious?

Reply:
I think since this is in HTTP library, there's nothing to gain for adding a term other than Body.